src/axom/core/Macros.hpp

```diff
@@ -197,8 +197,8 @@
  * \endcode
  */
 #define DISABLE_COPY_AND_ASSIGNMENT(className)                                 \
-  className( const className & ) = delete;                                     \
-  className& operator=(const className&) = delete
+  className( className & ) = delete;                                     \
+  className& operator=( className& ) = delete
```

Comment:
What was the issue with `const` there? Just curious?

Reply:
Ah, this is a mistake on my part. `const` is fine here.  I meant to remove const from DISABLE_MOVE_AND_ASSIGNMENT to match the syntax on[ cppreference](https://en.cppreference.com/w/cpp/language/move_assignment).